services/QuillLMS/client/app/bundles/Teacher/containers/UnitTemplates/unitTemplateRow.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react'
+
+import UnitTemplateActivityRow from './unitTemplateActivityRow';
+
+import { FlagDropdown } from '../../../Shared/index';
+
+
+const UnitTemplateRow = ({
+  handleDelete,
+  unitTemplate,
+  updateUnitTemplate
+}) => {
+  const [showActivities, setShowActivities] = React.useState<boolean>(false);
+
+  function deleteUnitTemplate() {
+    const confirm = window.confirm('Are you sure you want to delete ' + unitTemplate.name + '?');
+    if (confirm) {
+      handleDelete(unitTemplate.id);
+    }
+  }
+
+  function handleHideActivitiesRow() {
+    setShowActivities(!showActivities)
+  }
+
+  function handleSelectFlag(e) {
+    const { target } = e;
+    const { value } = target;
+    const newUnitTemplate = {...unitTemplate}
+    newUnitTemplate.flag = value
+    updateUnitTemplate(newUnitTemplate)
+  }
+
+  function handleRemoveActivity(act_id) {
+    const confirm = window.confirm('Are you sure you want to remove this activity?');
+    if (confirm) {
+      const newUnitTemplate = {...unitTemplate}
+      let activityIds = unitTemplate.activities.map((a) => a.id)
+      activityIds.splice(activityIds.indexOf(act_id), 1)
+      newUnitTemplate.activity_ids = activityIds
+      updateUnitTemplate(newUnitTemplate)
+    }
+  }
+
+  function expandOrCollapseButton() {
+    const imageLink = showActivities ? 'collapse.svg' : 'expand.svg'
+    const innerElement = <img alt="expand-and-collapse" src={`https://assets.quill.org/images/shared/${imageLink}`} />
```

Comment:
If we're going to have an alt, we should have it describe what it's doing - so like the `imageLink` we should make it a variable and have it be either "Expand" or "Collapse".